lib/routes/hitcon/zeroday.ts

```diff
@@ -0,0 +1,101 @@
+import type { Context } from 'hono';
+import { load } from 'cheerio';
+import puppeteer from '@/utils/puppeteer';
+import logger from '@/utils/logger';
+
+export const route: Route = {
+    name: '漏洞',
+    categories: ['programming'],
+    path: '/zeroday/vulnerability/:status?',
+    example: '/hitcon/zeroday/vulnerability',
+    parameters: {
+        status: '漏洞状态，见下表',
+    },
+    maintainers: ['KarasuShin'],
+    radar: [
+        {
+            source: ['zeroday.hitcon.org/vulnerability/:status?'],
+        },
+    ],
+    features: {
+        requirePuppeteer: true,
+    },
+    handler,
+    description: `| 缺省 | all | closed | disclosed | patching |
+  | ----- | ----- | ----- | ----- | ----- |
+  |  活動中 | 全部 | 關閉 | 公開 | 修補中 |`,
+};
+
+const baseUrl = 'https://zeroday.hitcon.org/vulnerability';
+
+const titleMap = {
+    all: '全部',
+    closed: '關閉',
+    disclosed: '公開',
+    patching: '修補中',
+};
+
+async function handler(ctx: Context): Promise<Data> {
+    let url = baseUrl;
+    const status = ctx.req.param('status');
+    if (status) {
+        url += `/${status}`;
+    }
+
+    const browser = await puppeteer();
+    const page = await browser.newPage();
+    await page.setRequestInterception(true);
+
+    page.on('request', (request) => {
+        request.resourceType() === 'document' || request.resourceType() === 'script' || request.resourceType() === 'xhr' ? request.continue() : request.abort();
+    });
+
+    logger.http(`Requesting ${url}`);
+    await page.goto(url, {
+        waitUntil: 'domcontentloaded',
+    });
+
+    const response = await page.evaluate(() => document.documentElement.innerHTML);
+    browser.close();
+
+    const $ = load(response);
+    const items = $('.zdui-strip-list>li')
+        .toArray()
+        .map((el) => {
+            const title = $(el).find('.title a');
+            const vulData = $(el).find('.vul-data');
+            const code = vulData
+                .find('.code')
+                .contents()
+                .filter(function () {
+                    return this.nodeType === 3;
+                })
+                .text();
+            const risk = vulData.find('.risk span').eq(1).text();
+            const vender = vulData.find('.vender').find('.v-name-full').text();
+            const status = vulData.find('.status').text().replace('Status:', '').trim();
+            const date = vulData.find('.date').text().replace('Date:', '').trim();
+            const reporter = vulData.find('.zdui-author-badge').find('a>span').text();
+
+            return {
+                title: title.text(),
+                link: title.attr('href'),
+                description: `<ul>
+                <li>${vender}</li>
+                <li>ZDID: ${code}</li>
+                <li>風險: ${risk}</li>
+                <li>處理狀態: ${status}</li>
+                <li>通報者: ${reporter}</li>
+                <li>通報日期: ${date}</li>
+                </ul>`,
+            };
```

Comment:
Please add `pubDate` and use [`art-template`](https://docs.rsshub.app/joinus/advanced/script-standard#rendering-templates) for rendering